internal/testing/cluster.go

```diff
@@ -566,18 +585,43 @@ func (c *TestCluster) Sync() error {
 	return c.sync(c.hosts)
 }
 
+// waitForHostAccounts will fetch the accounts from the worker and wait until
+// they have money in them
+func (c *TestCluster) waitForHostAccounts(hosts map[types.PublicKey]struct{}) error {
+	return Retry(30, time.Second, func() error {
+		accounts, err := c.Worker.Accounts(context.Background())
```

Comment:
Unless you changed it, accounts won't show up before you have tried to fund them for the first time.

Reply:
If you have a cluster with funding you should be good though right? If you are using a cluster with no funding you as a developer are purposefully using one without funding, so if you then time out here waiting for accounts I figure you should realise you can't wait for accounts if you're using a non-funded cluster...